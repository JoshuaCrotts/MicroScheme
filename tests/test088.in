(define (f l) 
    (if (eq? (car (car (cdr (car (cdr l))))) '+) "Yes" "No") 
)

(define l '(+ (* (+ 2 x) (+ 6 x) (* x 23))))
(cdr l)
(car (cdr l))
(cdr (car (cdr l)))
(car (cdr (car (cdr l))))
(car (car (cdr (car (cdr l)))))
(f l)
