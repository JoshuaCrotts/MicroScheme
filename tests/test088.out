((* (+ 2 x) (+ 6 x) (* x 23)))
(* (+ 2 x) (+ 6 x) (* x 23))
((+ 2 x) (+ 6 x) (* x 23))
(+ 2 x)
+
Yes
